packages/vscode-extension/src/builders/PlatformBuildCache.ts

```diff
@@ -53,21 +54,34 @@ export class PlatformBuildCache {
   /**
    * Passed fingerprint should be calculated at the time build is started.
    */
-  public async storeBuild(buildFingerprint: string, build: BuildResult) {
+  public async storeCache(buildFingerprint: string, build: BuildResult) {
```

Comment:
why has this been renamed? The method takes fingerprint and build result that gets stored. We don't store any cache here